test/test_provisioning_api.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+
+from mock import patch
+from urllib3 import disable_warnings
+
+import cloudinary.provisioning.account
+from test.helper_test import get_params, provisioning_response_mock
+
+MOCK_RESPONSE = provisioning_response_mock()
+
+disable_warnings()
+
+ACCOUNT_ID = "123415-s1234gdfs-341fd-aga-asdfasdgfasd"
+API_KEY = "123415-s1234gdfs-341fd-aga-asdfasdgfasd"
+API_SECRET = "123415-s1234gdfs-341fd-aga-asdfasdgfasd"
+
+SUB_ACCOUNT_NAME = "test_subaccount"
+SUB_ACCOUNT_ID = "123415-s1234gdfs-341fd-aga-asdfasdgfasd"
+
+NAME = "TestName"
+EMAIL = "test@email.com"
+USER_ID = "123415-s1234gdfs-341fd-aga-asdfasdgfasd"
+
+CLOUD_NAME = "cloud2019"
+
+
+class AccountApiTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cloudinary.provisioning.account.account_config(**dict(
+            account_id=ACCOUNT_ID,
+            provisioning_api_key=API_KEY,
+            provisioning_api_secret=API_SECRET
+        ))
+
+    @classmethod
+    def tearDownClass(cls):
+        return
+
+    @patch('urllib3.request.RequestMethods.request')
+    def test_sub_accounts_params(self, mocker):
+        """ should allow ids, enabled, prefix """
+        mocker.return_value = MOCK_RESPONSE
+
+        cloudinary.provisioning.account.sub_accounts(ids=1, enabled=True)
+
+        params = get_params(mocker.call_args[0])
```

Comment:
this line repeats multiple times, maybe refactor it? like: `params = get_mocker_params(mocker)`